qt/component.h

```diff
@@ -138,20 +178,46 @@ Q_GADGET
          * \return provided items for this \param kind
          */
         AppStream::Provided provided(Provided::Kind kind) const;
+        void addProvided(const AppStream::Provided& provided);
 
         QList<AppStream::Screenshot> screenshots() const;
+        void addScreenshot(const AppStream::Screenshot& screenshot);
 
         QList<AppStream::Release> releases() const;
+        void addRelease(const AppStream::Release& release);
 
+        bool hasBundle() const;
         QList<AppStream::Bundle> bundles() const;
         AppStream::Bundle bundle(Bundle::Kind kind) const;
+        void addBundle(const AppStream::Bundle& bundle) const;
 
         QList<AppStream::Suggested> suggested() const;
+        void addSuggested(const AppStream::Suggested& suggested);
+
+        QStringList searchTokens() const;
+        uint searchMatches(const QString& term) const;
+        uint searchMatchesAll(const QStringList& terms) const;
+
+        MergeKind mergeKind() const;
+        void setMergeKind(MergeKind kind);
 
+        QHash<QString,QString> custom() const;
+        QString customValue(const QString& key);
+        bool insertCustomValue(const QString& key, const QString& value);
+
+        /**
+         * TODO contentRating, launchable and languages
```

Comment:
Fixed, sorry, I haven't seen the error before. I saw CI failure at the beginning but there was a different problem (something with downloading or so) and I ignored it since then.